admin/models/pupil-restart.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const Schema = mongoose.Schema
+
+const PupilRestart = new Schema({
+  pupilId: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    ref: 'Pupil'
+  },
+  recordedByUser: {
```

Comment:
just so the others are aware - this will store the username.  Its a temporary fixture until we move to SQL and we will then use formal foreign keys by unique id